bubblesort.rb

```diff
@@ -0,0 +1,27 @@
+def bubble_sort(array)
+  switch = true
+  while switch
+    switch = false 
+    for i in (0...array.length - 1)
+      if array[i] > array[i + 1]
+        switch = true 
+        array[i], array[i + 1] = array[i + 1], array[i]
+      end
+    end
+  end  
+  array 
+end
+
+def bubble_sort_by(array) 

```

Comment:
Hi, we seem to run into a bump here in the bubble sort by method, did you try running the example provided by The Odin Project, [HERE](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks), because if I run 

`bubble_sort_by(["hi","hello","hey"]) do |left,right| `
  >   `left.length - right.length`
  > `end `

I do not get a value in return, or a sorted array, please do the necessary adjustments so that the method itself is capable of handling the example above, because so far all it returns is nil.

Reply:
Done